playgame.py

```diff
@@ -2,9 +2,15 @@
 import time
 from pydoc import locate
 from tools.hanabi_table import HanabiTable
+from tools.hanabi_card import HanabiColor
 import sys
 
 def main(argv):
+    args = parse_args()
+    game = HanabiGame(args)
+    game.play_game(args)
```

Comment:
It seems sort of weird to need to pass in `args` here while also passing them to the constructor.

Do you think it would be weird for `HanabiGame` to know about `verbose`? It looks like the verbosity setting is the only thing `play_game` needs `args` for at this time.